### ISSUE TITLE
Cannot use spyder after update

### ISSUE THREAD

**Issue by bingcao2** [flatironinstitute/CaImAn#391]
Hi All,
Thank you for your efforts to update CaImAn. I have updated it on my mac (High Sierra, 10.13.6) however could not run the spyder as I used to before. My caiman is installed in anaconda with python 3.6. I could use Jupyter to successfully run the demo in notebook, which means the code works 
well in my mac. Please find the error log when I try to run spyder as follows. Additionally, I also chose some previous version of spyder, such as 3.2.1 or above, and experienced same error.

My aim is to copy all of ∆F/F from the workspace of spyder for further analysis. If you have any solution to work it out in other IDE, such as pycharm, I am happy to learn it. 
If you need me supply more information, please let me know.

---------------error log--------------------------------

> Traceback (most recent call last): File "/anaconda3/envs/caiman/lib/python3.6/site-packages/qtpy/QtWebEngineWidgets.py", line 22, in  from PyQt5.QtWebEngineWidgets import QWebEnginePage ModuleNotFoundError: No module named 'PyQt5.QtWebEngineWidgets'  During handling of the above exception, another exception occurred:  Traceback (most recent call last): File "/anaconda3/envs/caiman/bin/spyder", line 11, in  sys.exit(main()) File "/anaconda3/envs/caiman/lib/python3.6/site-packages/spyder/app/start.py", line 189, in main from spyder.app import mainwindow File "/anaconda3/envs/caiman/lib/python3.6/site-packages/spyder/app/mainwindow.py", line 92, in  from qtpy import QtWebEngineWidgets # analysis:ignore File "/anaconda3/envs/caiman/lib/python3.6/site-packages/qtpy/QtWebEngineWidgets.py", line 26, in  from PyQt5.QtWebKitWidgets import QWebPage as QWebEnginePage ModuleNotFoundError: No module named 'PyQt5.QtWebKitWidgets'

**Comment by epnev** [comment]
@bingcao2 Does this happen on all the files or a specific demo?
Please also list the output of 
```
conda list pyqt
```

**Comment by bingcao2** [comment]
> @bingcao2 Does this happen on all the files or a specific demo?
> Please also list the output of
> 
> ```
> conda list pyqt
> ```
Thanks for your reply.
This happens when I try to run Spyder in the caiman, not related to demo or other files.
Please find the information of pyqt as follows.
```
(caiman) bash-3.2$ conda list pyqt
# packages in environment at /anaconda3/envs/caiman:
#
# Name                    Version                   Build  Channel
pyqt                      5.6.0            py36h8210e8a_7    conda-forge
```

**Comment by agiovann** [comment]
@pgunn is this something you bumped into before?

**Comment by pgunn** [comment]
Hi,
Can you try the following command?
`conda install -f qt`

**Comment by bingcao2** [comment]
> Things I've looked at to reach this suggestion:
> [ContinuumIO/anaconda-issues#9199](https://github.com/ContinuumIO/anaconda-issues/issues/9199)
> [conda-forge/pyqt-feedstock#19](https://github.com/conda-forge/pyqt-feedstock/issues/19)

Thanks for the information. I use `conda install qt=5.9` to fix the spyder problem.
However, I experienced errors when I tried to run any of our demos. Please find the error logs as follows. I can run the demo in notebook perfectly using Jupyter, so I think my installment of CaImAn is correct. 
Many thanks for your help!

```
The autoreload extension is already loaded. To reload it, use:
  %reload_ext autoreload
Traceback (most recent call last):

  File "<ipython-input-1-e99364fcab63>", line 31, in <module>
    import caiman as cm

  File "/Users/Bing/CaImAn/caiman/__init__.py", line 3, in <module>
    from .base.movies import movie, load, load_movie_chain

  File "/Users/Bing/CaImAn/caiman/base/movies.py", line 67, in <module>
    from ..utils import visualization

  File "/Users/Bing/CaImAn/caiman/utils/visualization.py", line 34, in <module>
    from ..summary_images import local_correlations

  File "/Users/Bing/CaImAn/caiman/summary_images.py", line 31, in <module>
    from caiman.source_extraction.cnmf.pre_processing import get_noise_fft

  File "/Users/Bing/CaImAn/caiman/source_extraction/cnmf/__init__.py", line 12, in <module>
    from . import oasis

ImportError: cannot import name 'oasis'
```

**Comment by agiovann** [comment]
@bingcao2 maybe is it more complex than this, but try to recompile oasis (reinstall caiman with pip install .)

**Comment by j-friedrich** [comment]
I think it's a different error. The ImportError is indeed most likely due to not compiling oasis.

**Comment by bingcao2** [comment]
> @bingcao2 maybe is it more complex than this, but try to recompile oasis (reinstall caiman with pip install .)

Hi All,

I find that only updating qt to 5.9 is not a good solution for the spyder issue. I could run spyder after updating qt, however, my jupyter and spyder experienced a lot of errors which never happened when qt=5.6, even though I have reinstalled caiman. So I have installed caiman through pycharm and change the python interpreter, which works very well right now. Many thanks to all of you for helping me fix this issue. 

By the way, as my final aim is to extract the ∆F/F, could you tell me the usually variable name you used that I can find in the pycharm variables? To compare with the workspace in spyder, it is not easy to figure out the correct variable in the pycharm.

Thanks again.

Bing

**Comment by epnev** [comment]
@bingcao2 The DF/F traces are included in `cnm.estimates.F_dff`. If you want to store them separately you can use some numpy command, e.g., [`np.savez`](https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.savez.html)
Is that what you're looking for?

**Comment by agiovann** [comment]
@bingcao2 there is also a scientific mode in pycharm that will allow you to use the code interactively like in spyder https://www.jetbrains.com/help/pycharm/matplotlib-tutorial.html

**Comment by pgunn** [comment]
@bingcao2 FYI, I found a less disruptive fix for this issue in #409

**Comment by bingcao2** [comment]
Thank you Pat! I will read it. 
Bing
